stub-environment/hadoop_cluster.txt

```diff
@@ -1,4 +1,4 @@
 test-laptop 08:00:27:56:A2:21 10.0.100.5 - bcpc_host bcpc.example.com SKIP
 bcpc-vm1 08:00:27:56:A2:28 10.0.100.11 - bcpc_host_trusty bcpc.example.com role[BCPC-Hadoop-Head],role[BCPC-Hadoop-Head-Namenode],role[BCPC-Hadoop-Head-HBase],recipe[bcpc-hadoop::copylog]
-bcpc-vm2 08:00:27:E5:3A:00 10.0.100.12 - bcpc_host_trusty bcpc.example.com role[BCPC-Hadoop-Head],role[BCPC-Hadoop-Head-ResourceManager],role[BCPC-Hadoop-Head-Namenode-Standby],role[BCPC-Hadoop-Head-MapReduce],role[BCPC-Hadoop-Head-Hive],recipe[bcpc-hadoop::copylog],role[BCPC-Hadoop-Head-YarnTimeLineServer]
+bcpc-vm2 08:00:27:E5:3A:00 10.0.100.12 - bcpc_host_trusty bcpc.example.com role[BCPC-Hadoop-Head],role[BCPC-Hadoop-Head-ResourceManager],role[BCPC-Hadoop-Head-Namenode-Standby],role[BCPC-Hadoop-Head-MapReduce],role[BCPC-Hadoop-Head-Hive],recipe[bcpc-hadoop::copylog]
```

Comment:
Can you add the nodeids here as well?

`stub-environment/kafka_cluster.txt` would need to be updated as well.